TheWarWithin/DemonHunterVengeance.lua

```diff
@@ -1559,7 +1606,7 @@ spec:RegisterAbilities( {
 
         startsCombat = false,
         texture = 1344652,
-        nobuff = function () return talent.demonic_intensity.enabled and "metamorphosis" or nil end,
+        nobuff = function() if talent.demonsurge.enabled then return "demonsurge_demonic" end end,
```

Comment:
The returned buff should be `demonsurge_hardcast` since Sigil of Flame is only empowered if talented into Demonic Intensity. However, I think it's simpler if the whole function is just replaced with:
```
nobuff = "demonsurge_hardcast",
```
This will work regardless of the Hero Talents chosen.